order/command-side/build.gradle

```diff
@@ -2,6 +2,7 @@ plugins {
     id 'com.google.cloud.tools.jib'
 }
 jib.to.image = "noahhsu/espoc/order/${project.name}:${version}"
+jib.to.tags = ["latest"]
```

Comment:
i think put this attr. in the build.gradle file of root project may work,

sth like
```
subprojects {

 jib.to.tag= ["latest", "${version}"]
}
```

Reply:
I think I've tried something like that but gradle was complaining not knowing what jib is. I think it's because jib plugin only applied to the "leaf" projects. I'll try it again sometime.

Reply:
i've try te below method and it works, you can take a look

root build.gradle
``` 
subprojects {
    apply plugin: 'com.google.cloud.tools.jib'

    jib.to.tags = ["latest"]
```
non app module (depend on which task you use)
```
jib {
    enabled = false
}
jibDockerBuild {
    enabled = false
}
```

leaf module (spring boot app module)
<img width="454" alt="image" src="https://user-images.githubusercontent.com/58896446/210384436-b10aab0c-7297-4dcd-931d-18e81d47046d.png">
----
and the result becomes
<img width="678" alt="image" src="https://user-images.githubusercontent.com/58896446/210384766-4422edbe-8712-4b14-9251-e0dd48da8f98.png">

Reply:
I failed like this 
![image](https://user-images.githubusercontent.com/22278616/218235874-bbfbd676-b1db-4418-af6d-33b2de421cd1.png)

Reply:
according to this issue, maybe we should put the plugin setting in gradle.build of the second layer module (i.e. order, shipment, payment) instead of the root module
ref: https://github.com/GoogleContainerTools/jib/issues/3537

Reply:
it seems like we can only set jib to false by `tasks.findByName("jib").setEnabled(false)`

there is some bug for `jib.enabled = false`